service/src/main/java/com/codeforcommunity/requester/S3Requester.java

```diff
@@ -325,4 +327,23 @@ public static LoadTemplateResponse loadHTML(String name, String directoryName) {
 
     return new LoadTemplateResponse(HTMLContent, HTMLFile.getKey(), htmlAuthor);
   }
+
+  /**
+   * Delete the existing HTML file with the given name from the user uploads S3 bucket.
+   *
+   * @param name the name of the HTML file in S3 to be deleted.
+   * @param directoryName the directory of the file in S3 (without leading or trailing '/').
+   * @throws InvalidURLException if the file does not exist.
+   * @throws SdkClientException if the deletion from S3 failed.
+   */
+  public static void deleteHTML(String name, String directoryName) {
+    String HTMLPath = directoryName + "/" + name + "_template.html";
+
+    Boolean htmlExists = externs.getS3Client().doesObjectExist(externs.getBucketPublic(), HTMLPath);
+
```

Comment:
🔧 Can we use the `pathExists()` method here as well? We should also throw an `InvalidURLException` if the file doesn't exist